packages/frontend/src/ui/select/styles.module.css

```diff
@@ -0,0 +1,37 @@
+.input :global(.input) {
+    @apply cursor-pointer;
+}
+
+.input :global(.inputIcon) {
+    @apply w-4;
+}
+
+.dropdownRoot {
+    @apply max-h-48
+        z-20
+        overflow-hidden
+        h-fit;
+}
+
+.emptyOptionList {
+    @apply flex
+        justify-center
+        items-center
+        p-3
+        h-12;
+}
+
+.option {
+    @apply cursor-pointer
+        p-3
```

Comment:
Add the usual transition classes (duration, ease, ...).

Reply:
Addressed in aa2e218191c5a74d76060d9f84949c7e9ac1e843